### PR TITLE
Fix study name for CBTTC Atlas

### PR DESCRIPTION
**Pull request by jecos**

### CODE REVIEW

File: src/main/resources/flyway/sql/V4__rename_study_cbttc_atlas.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * This is done so that the former queries remain valid once the studies have been renamed in the release coordinator.
+ * 
+ * @date 2019-08-07
+ */
+
+/*
+ * The search queries in the cohort builder are stored as JSON.
+ * We include the initial double quote, to reduce risks of false positives.
+ */
+UPDATE riff SET content = REPLACE(content::TEXT, '"Consortium: Pediatric Brain Tumors - CBTTC', '"Pediatric Brain Tumor Atlas: CBTTC')::JSON;
```
Comment (jberube) [inline src/main/resources/flyway/sql/V4__rename_study_cbttc_atlas.sql]: Is the "quote and double-quote" voluntary?

`'"Consortium: `

Comment (jberube) [reply src/main/resources/flyway/sql/V4__rename_study_cbttc_atlas.sql]: Nevermind, I've understood why.

File: src/main/resources/flyway/sql/V4__rename_study_cbttc_atlas.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+/*
+ * The search queries in the file repository are stored as their full URL. The SQON is therefore URL-encoded:
+ * We include the initial double quote, to reduce risks of false positives.
+ */
+UPDATE riff SET content = REPLACE(content::TEXT, '%22Consortium3A%20Pediatric%20Brain%20Tumors%20-%20CBTTC', '%22Pediatric%20Brain%20Tumors%20Atlas%3A%20CBTTC')::JSON;
```
Comment (jberube) [inline src/main/resources/flyway/sql/V4__rename_study_cbttc_atlas.sql]: `%` missing before `3A` in `'%22Consortium3A`

Comment (jecos) [reply src/main/resources/flyway/sql/V4__rename_study_cbttc_atlas.sql]: No, check the previous PR. It was omitted. => https://github.com/kids-first/riff/pull/15

**Pull request merged**